src/main/resources/epub/chapter_template.xhtml

```diff
@@ -0,0 +1,7 @@
+<html xmlns="http://www.w3.org/1999/xhtml">
+<head>
+    <title>{0}</title>
+    <link href="epub.css" rel="stylesheet" type="text/css"/>
```

Comment:
Yes, it actually does work, but it was a bit error-prone solution.
I have rewritten the logic of applying the templates and created case classes for this instead of reading from a template file. The templates are so simple, that it is better (at least for now) to only have them as constants.